docs/index.md

```diff
@@ -19,8 +19,7 @@ This section describes how to:
 
 Before you install TSC, confirm that you have the following dependencies installed:
 
-* Python. You can use TSC with Python 2.7.9 or later and with Python 3.3 or later. These versions include pip, which is
-  the recommended means of installing TSC.
+* Python. You can use TSC with Python 3.3 or later. 
```

Comment:
We stopped validating against 3.3 (.travis.yml file), so maybe we should say 3.5+?